docs/tool_jextract.md

```diff
@@ -30,14 +30,21 @@ On some operating systems, copies of executable files and libraries are required
 
 ## Syntax
 
-        jextract <core file name> [<zip_file>]
+        jextract [-r] <core file name> [<zip_file>]
 
 where:
 
+- `-r` forces the `jextract` utility to proceed when the system dump is created from an SDK with a different build ID. See **Restriction**.
 - `<core file name>` is the name of the system dump.
 - `<zip_file>` is the name you want to give to the processed file. If you do not specify a name, output is written to `<core file name>.zip` by default. The output is written to the same directory as the core file.
 
-: <i class="fa fa-exclamation-triangle"></i> **Restriction: ** You must run `jextract` on the same system that produced the system dump in order to collect the correct executables and libraries referenced in the system dump. You should also run `jextract` using the same VM level, to avoid any problems.
+: <i class="fa fa-exclamation-triangle"></i> **Restriction:** You should run `jextract` on the same system that produced the system dump in order to collect the correct executables and libraries referenced in the system dump. You should also run `jextract` using the same VM level, to avoid any problems. From Eclipse OpenJ9 V0.24.0, the utility always checks that the build ID of the SDK that created the dump file matches the `jextract` build ID. Where these IDs do not match, the following exception is thrown:
+
+    ```
+    J9RAS.buildID is incorrect (found XXX, expecting YYY). This version of jextract is incompatible with this dump.
+    ```
+
+    To continue, despite the mismatch, use the `-r` option.
```

Comment:
There are no back-quotes in the message and it appears all on one line:
```
J9RAS.buildID is incorrect (found XXX, expecting YYY). This version of jextract is incompatible with this dump (use '-r' option to relax this check).
```

Reply:
Line 47 is redundant.

Reply:
Yes it is, unless you fail to read all of the exception message, which I think, will be scrolled out of view in the built files. I'll be staging this to check.

Reply:
Fair enough, but it should not have the same formatting as the (someone) literal message; I'll have another look at the staging site when it reappears.

Reply:
The result looks good, except for the absence of that scrollbar you anticipated: Can you arrange for a scrollbar?

Reply:
Scroll bars should be automatic .... it'll be a markup issue with the bulleted list I suspect. I'll keep going at it until I get it work properly. 

BTW, what did you mean by "It should not have the same formatting as the (someone) literal message"?? Not clear what you want here.

Reply:
I meant to write 'somewhat' instead of 'someone', meaning that the example is almost exactly what the message would say. The example is formatted differently than normal text, the comment that follows should use the normal font (it now does). Thank you.

Using chrome, I don't see any horizontal scrollbars at https://eclipse.github.io/openj9-docs-staging/650/tool_jextract/, nor with firefox.

Reply:
Me neither ... still waiting for it to stage. A side effect of adding all that API doc is that we've slowed to a crawl. (That's on my to do list - to try and find a solution)

Reply:
My last attempt with a fix for today @keithc-ca but I'll come back to this if it doesn't work. Bit painful, but I'll get there.

Reply:
I see a scrollbar now. Thanks.